src/app/features/genes/components/gene-hero/gene-hero.component.html

```diff
@@ -14,17 +14,31 @@ <h2 class="gene-hero-name">
       <p class="gene-hero-provider">
         {{ getSummary() }}
       </p>
-      <ng-container *ngIf="gene.bio_domains || getAlias()">
+      <ng-container *ngIf="gene.bio_domains || getAlias() || getEnsemblUrl() !== ''">
         <hr />
         <div *ngIf="gene.bio_domains" class="gene-hero-biodomains">
           <h4 class="gene-hero-biodomains-heading">Biological Domains</h4>
           <p>
             {{ getBiodomains() }}
           </p>
         </div>
-        <div *ngIf="getAlias()" class="gene-hero-aliases">
+        <div *ngIf="getAlias() || getEnsemblUrl() !== ''" class="gene-hero-aliases">
```

Comment:
Per the ticket, "The AKA header and the first line underneath (which displays the current gene details page’s ENSG)  it should always be displayed." Should the conditional be removed from this div?

Reply:
you're right.  it is redundant.